skymp5-server/cpp/server_guest_lib/PieScript.h

```diff
@@ -24,16 +26,35 @@ class PieScript
     Tier5
   };
 
-  using LootTable =
-    std::unordered_map<LootboxItemType,
-                       std::unordered_map<Tier, std::vector<uint32_t>>>;
+  enum class StarterKitType
+  {
+    ChefKit,
+    MinerKit,
+    PrisonerKit,
+    LumberjackKit,
+    PatronKit
+  };
+
+private:
+  enum StarterKitChance
+  {
+    ChefKitChance = 25,
+    MinerKitChance = 25,
+    PrisonerKitChance = 25,
+    LumberjackKitChance = 25
+  };
 
 public:
   PieScript(std::vector<std::string> espmFiles);
 
 public:
+  using LootTable =
+    std::unordered_map<LootboxItemType,
+                       std::unordered_map<Tier, std::vector<uint32_t>>>;
   const LootTable& GetLootTable() const;
   void Play(MpActor* actor);
+  void GetStarterKitItems(MpActor* actor);
```

Comment:
Use this if possible
```suggestion
  void GetStarterKitItems(const MpActor& actor);
```